app/services/idv/session.rb

```diff
@@ -106,7 +106,8 @@ def clear
     end
 
     def associate_in_person_enrollment_with_profile
-      return unless pending_in_person_enrollment? && current_user.establishing_in_person_enrollment
+      return unless current_user.has_in_person_enrollment?
```

Comment:
```suggestion
      return unless has_in_person_enrollment?
```
To take advantage of your method